packages/discord.js/src/structures/Presence.js

```diff
@@ -389,6 +391,17 @@ class RichPresenceAssets {
 
     return this.activity.presence.client.rest.cdn.appAsset(this.activity.applicationId, this.largeImage, options);
   }
+
+  /**
+   * Tests whether an image key is a URL or not
+   * @private
+   * @param {string} imageKey The image key to test
+   * @returns {boolean}
```

Comment:
Usually `@private` is the last tag, also looks nicer
```suggestion
   * @param {string} imageKey The image key to test
   * @returns {boolean}
   * @private
```